client/routes.js

```diff
@@ -40,6 +43,8 @@ class Routes extends Component {
           <Switch>
             <Route exact path="/product/add" component={AddProduct} />
             <Route path="/product/:productId/edit" component={EditProduct} />
+            <Route path="/review/add" component={(AddReview)} />
+            {/* <Route path="/review/:reviewId/edit" component={EditReview} /> */}
```

Comment:
The edit route is disabled here.  Let's fix editing before we merge into master.